# The message defines the 2D bounding box of a certain object in the image.

# ID or label of the detected object
string object_id

# 2D coordinatses of the bounding box in the image plane
int32 x_min
int32 y_min
int32 x_max
int32 y_max